tfstate/remote_azurerm.go

```diff
@@ -121,3 +133,24 @@ func getDefaultAccessKey(ctx context.Context, resourceGroupName string, accountN
 
 	return *keys.Keys[0].Value, nil
 }
+
+func getSubscription(opt azureRMOption) (string, error) {
+	if opt.subscriptionID != "" {
+		return opt.subscriptionID, nil
+	}
+
+	subscriptionID, err := getDefaultSubscription()
+	if err != nil {
+		return "", errors.Wrap(err, "failed to get default subscription")
+	}
+
+	return subscriptionID, nil
+}
+
+func getDefaultClient() (*azidentity.DefaultAzureCredential, error) {
```

Comment:
```suggestion
func getDefaultAzureCredential() (*azidentity.DefaultAzureCredential, error) {
```

This function does not return "Client", seems to return "Credential".
Would you consider renaming the function?

Reply:
Done!